### PR TITLE
Create group functionality implemented.

### PR DESCRIPTION
**Pull request by malloryporche**
### components/Group/CreateGroupModal

- Added ability to create new groups

### pages/group/index.tsx

- Added modal button to main group page

### CODE REVIEW

File: src/components/Group/CreateGroupModal.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            setDisplayLoginModal(true);
+          }
+        },
+        onSuccess() {
+          setGroupDetails({
+            ...groupDetails,
+            name: "",
+            description: "",
+          });
+          console.log(groupDetails);
```
Comment (redconOne) [inline src/components/Group/CreateGroupModal.tsx:52]: Can we remove this log to keep the console a little more clear?

File: src/pages/group/index.tsx [hunk 2]
```diff
@@ -13,15 +14,28 @@ export default function GroupsSearch() {
...
         {groupsList.data?.groups.map((group) => (
-          <div key={group.id} className="flex ring-base-500 rounded-lg bg-base-100 ring-1 p-3">
-            <Link href={`/${group.id}`} className="relative h-10 w-10 overflow-clip rounded-lg bg-secondary ring-1 ring-base-200">
-              <Image src={`${group.photoUrl}`} alt={`Image for ${group.name}`} width={50} height={50} unoptimized={true}/>
+          <div
+            key={group.id}
+            className="flex rounded-lg bg-base-100 p-3 ring-1 ring-base-500"
+          >
+            <Link
+              href={`/${group.id}`}
```
Comment (redconOne) [inline src/pages/group/index.tsx]: Is this linking where intended? Not sure if we're wanting to point to `/group/${group.id}` or `/${group.id}`.

Comment (malloryporche) [reply src/pages/group/index.tsx]: @redconOne I believe this is already being addressed in Derek's PR he submitted last night.  Not sure how that works if I change it after his PR is approved?

Comment (burhan-syed) [reply src/pages/group/index.tsx]: should be `/group/${group.id}`

Comment (malloryporche) [reply src/pages/group/index.tsx]: Updated.

**Review by redconOne — APPROVED** [review]
Looks good. Thanks for figuring out that link issue.

**Pull request merged**